config/domains.php

```diff
@@ -1,17 +1,17 @@
 <?php
-Configure::set('DomainManager.first_reminder_days_min', 26);
-Configure::set('DomainManager.first_reminder_days_max', 35);
-Configure::set('DomainManager.second_reminder_days_min', 4);
-Configure::set('DomainManager.second_reminder_days_max', 10);
-Configure::set('DomainManager.expiry_notice_days_min', 1);
-Configure::set('DomainManager.expiry_notice_days_max', 5);
+Configure::set('Domains.first_reminder_days_min', 26);
+Configure::set('Domains.first_reminder_days_max', 35);
+Configure::set('Domains.second_reminder_days_min', 4);
+Configure::set('Domains.second_reminder_days_max', 10);
+Configure::set('Domains.expiry_notice_days_min', 1);
+Configure::set('Domains.expiry_notice_days_max', 5);
 Configure::set(
-    'DomainManager.install.emails',
+    'Domains.install.emails',
     [
         [
-            'action' => 'DomainManager.domain_renewal_1',
+            'action' => 'Domains.domain_renewal_1',
             'type' => 'client',
-            'plugin_dir' => 'domain_manager',
+            'plugin_dir' => 'domains',
             'tags' => '{domain},{service},{contact},{client_uri}',
             'from' => 'sales@mydomain.com',
             'from_name' => 'Domain Manager',
```

Comment:
I think the name for the email templates should be "Domains" or "{company} Domains" replacing {company} by the name of the current company.

Reply:
Perhaps true, but not for this task.